src/components/regboks.tsx

```diff
@@ -0,0 +1,133 @@
+import { Panel, Heading } from '@navikt/ds-react';
+import { DobbeltInformasjon } from './felles/dobbelinfo';
+import { RegistreringsData } from '../data/api/datatyper/registreringsData';
+import { useEffect, useState } from 'react';
+import { hentRegistrering } from '../data/api/fetch';
+import { useAppStore } from '../stores/app-store';
+import { Errormelding, Laster } from './felles/minikomkomponenter';
+import { StringOrNothing } from '../utils/felles-typer';
+import EMDASH from '../utils/emdash';
+import { EnkeltInformasjon } from './felles/enkeltInfo';
+import { formaterDato } from '../utils/formater';
+
+export const Registrering = () => {
+    const { fnr } = useAppStore();
+    const [registrering, setRegistrering] = useState<RegistreringsData | null>(null);
+    const [lasterData, setLasterData] = useState<boolean>(true);
+    const [harFeil, setHarFeil] = useState<boolean>(false);
+
+    useEffect(() => {
+        const hentOverblikkData = async () => {
```

Comment:
Selv om koden er gjenbruk, så er det fint å gi den et nytt navn som gjelder for det den gjør i denne komponenten